main.py

```diff
@@ -1,10 +1,51 @@
 import init_django_orm  # noqa: F401
+import json
 
 from db.models import Race, Skill, Player, Guild
 
 
 def main() -> None:
-    pass
+
+    with open("players.json", "r") as players_file:
+        players = json.load(players_file)
+
+    for nickname, info in players.items():
+        guild = info.get("guild")
+        race = info.get("race")
+        skills = info["race"].get("skills")
+
+        if guild is not None:
+            if not Guild.objects.filter(name=guild["name"]).exists():
+                guild = Guild.objects.create(
+                    name=guild["name"],
+                    description=guild["description"]
+                )
+            else:
+                guild = Guild.objects.get(name=guild["name"])
```

Comment:
You can use get_or_create
https://docs.djangoproject.com/en/4.1/ref/models/querysets/#get-or-create